app/backend/src/clj/teet/file/file_db.clj

```diff
@@ -193,8 +193,8 @@
         next-version (fn [file]
                        (some #(when (= (:db/id file)
                                        (get-in % [:file/previous-version :db/id])) %)
-                             next-versions))]
-    (vec
+                             next-versions))
+    meta-files (vec
```

Comment:
fix indentation

Reply:
the name `meta-files` doesn't seem right... perhaps it should be moved out of let and the sorting just wrapped around the `for`

Reply:
It is not clear what is the defect here

Reply:
indentation, indent meta-files starting from the same column as other let bindings

the other is a comment about the name, which isn't clear to me... why are they "meta" files?